src/darwin/mod.rs

```diff
@@ -80,9 +81,9 @@ impl Descriptor {
             // many chars. We add the symbols back in.
             // Investigate if we can actually use the .-prefixed
             // fallbacks somehow.
-            if let Ok(apple_symbols) = new_from_name("Apple Symbols", size) {
-                fallbacks.push(Font { ct_font: apple_symbols, fallbacks: Vec::new() })
-            };
+            let name = CFString::from_static_str("Apple Symbols");
+            let apple_symbols = unsafe { CTFont::with_name(&name, size, ptr::null_mut()) };
+            fallbacks.push(Font { ct_font: apple_symbols, fallbacks: Vec::new() });
```

Comment:
What if the font doesn't exist? It looks like previously we were handling failure here.